### ISSUE TITLE
Proposal: Decrease the unit price of energy

### ISSUE THREAD

**Issue by Ademillery** [tronprotocol/tips#511]
# Simple Summary

Decrease the unit price of energy.

# Motivation

The energy unit price of the TRON network has been increased a couple of times during the last 2 years, making the substantial growth of the TRX staking ratio and deflation rate, the expected goal has basically been achieved.

The increase of energy unit price plays an important role in improving the security and stability of the TRON network, but also it increases the cost of using dApps at the same time. The [dynamic energy model](https://github.com/tronprotocol/tips/issues/508) is proposed by the community to strike low-value transactions without increasing the usage cost of other dApps, making space to decrease the unit price of energy. Therefore I suggest decreasing the unit price of energy according to the influence after the dynamic energy model takes effect, for example decreasing from 420 sun to 360 sun, to further keep the competitive advantage of the TRON network by reducing the usage cost of dApps.

# How to initiate the voting request

Change the unit price of energy to 360 sun  (wallet-cli)

- createProposal 11 360

# Background

From the figure below, we can see that in the past period of time, the staking amount and staking rate of the TRON network have greatly increased, and the TRX circulation has shown a clear downward trend,

<img width="1023" alt="image" src="https://user-images.githubusercontent.com/118337824/216221452-11d1a340-cc65-477b-9a5c-9ec06fba5aa7.png">

## Comparison

Comparison of TRON energy unit price and Ethereum gas fee:

Ethereum Gas Price | TRON Energy Price
-- | --
30 Gwei ($0.00005) | 420 sun ($0.00002）

As can be seen from the above table, the current development cost of TRON is much lower than that of Ethereum, so reducing the unit price of energy will further maintain the advantage of the low cost of the TRON network.

**Comment by ustx** [comment]
Good proposal. Will the SRs vote for it?

**Comment by Ademillery** [comment]
> Good proposal. Will the SRs vote for it?

I hope to get support from community and SRs

**Comment by ufmannk** [comment]
Interesting！

**Comment by Jackmrshen** [comment]
I did not expect to see this kind of proposal in tron in my lifetime. Even though this is a hedge proposal to the stupid dynamic model with no doubt, at least this one is a correct thing. So is it that the rising of other tokens in the market makes tron foundation jealous?  

We need more proposals like this to enjoy TRX. Someone please give more ideas to increase TRX price.

**Comment by jernganl** [comment]
Thanks, bro!!!

**Comment by Jamestepfoward** [comment]
Good but not enough, when increasing, it is 10 times, 2 times, but when decreasing, it is 6/7, I think we should make it clear, as the price adjustment is according to the dust transactions, staking rate, and deflation rate, can we make a target state value list? So we will know when to increase the price and when to decrease, rather than so much meaningless discussion.

**Comment by Jakevsky** [comment]
I agree with you, and I think this proposal intimates something. Previously I thought there would be endless price increases and finally today I see the inflection point.  Then I think maybe the status of the TRON network is coming to meet the expectation. Let’s see whether this state is a benchmark for the TRON to govern the chain.

**Comment by Jakevsky** [comment]
I agree with you, and I think this proposal intimates something. Previously I thought there would be endless price increase and finally today I see the inflection point.  Then I think maybe the status of the TRON network is coming to meet the expectation. Let’s see whether this state is a benchmark for the TRON to govern the chain.

**Comment by Ocea91** [comment]
This is definitely what the community needs. The cost of USDT transfer is doubled now.

**Comment by Ademillery** [comment]
> I did not expect to see this kind of proposal in tron in my lifetime. Even though this is a hedge proposal to the stupid dynamic model with no doubt, at least this one is a correct thing. So is it that the rising of other tokens in the market makes tron foundation jealous?
> 
> We need more proposals like this to enjoy TRX. Someone please give more ideas to increase TRX price.

I am a community developer, this proposal should get votes from SRs to take effect. Only that I think the performance of the TRON network is good enough now to support this proposal.

**Comment by Ademillery** [comment]
> I agree with you, and I think this proposal intimates something. Previously I thought there would be endless price increase and finally today I see the inflection point. Then I think maybe the status of the TRON network is coming to meet the expectation. Let’s see whether this state is a benchmark for the TRON to govern the chain.

Previous proposals mean that the community think the status is not good enough so the energy price and consumption amount increase. So I think it may not be good to go back straight, it would be a waste of time. But I think we can make adjustments by a series of small steps.

**Comment by andreujuanc** [comment]
The new dynamic model is actually just a form of onchain discrimination.

**Comment by Jamestepfoward** [comment]
> The new dynamic model is actually just a form of onchain discrimination.

The community and dApps in the ecosystem seems have accepted it, I do not want to admit but I can see decrease of low-value transactions in blocks. Waiting for further data analysis and waiting for more proposals like this.

**Comment by ikirudoughnuts** [comment]
Hey, just saw it, what's going on with this TIP actually? The fee of transactions on TRON is unbearably high now, almost same or even more expensive than Ethereum, especially the TRX price keeps increasing these months!

**Comment by Ademillery** [comment]
> Hey, just saw it, what's going on with this TIP actually? The fee of transactions on TRON is unbearably high now, almost same or even more expensive than Ethereum, especially the TRX price keeps increasing these months!

I think this TIP did not attract enough attention from community, but yes as the transaction fee keeps rising, it becomes more important and urgent to get back to this. Fortunately, as it's still tagged with discussion-to, guess this issue is not out of date. Actions should be made to keep TRON competitive.

**Comment by ikirudoughnuts** [comment]
> > Hey, just saw it, what's going on with this TIP actually? The fee of transactions on TRON is unbearably high now, almost same or even more expensive than Ethereum, especially the TRX price keeps increasing these months!
> 
> I think this TIP did not attract enough attention from community, but yes as the transaction fee keeps rising, it becomes more important and urgent to get back to this. Fortunately, as it's still tagged with discussion-to, guess this issue is not out of date. Actions should be made to keep TRON competitive.

Even this proposal is approved, it's still not enough, I expect it as cheap as before, the parameter also worth discussing.

**Comment by WalterBrooks** [comment]
> > > Hey, just saw it, what's going on with this TIP actually? The fee of transactions on TRON is unbearably high now, almost same or even more expensive than Ethereum, especially the TRX price keeps increasing these months!
> > 
> > 
> > I think this TIP did not attract enough attention from community, but yes as the transaction fee keeps rising, it becomes more important and urgent to get back to this. Fortunately, as it's still tagged with discussion-to, guess this issue is not out of date. Actions should be made to keep TRON competitive.
> 
> Even this proposal is approved, it's still not enough, I expect it as cheap as before, the parameter also worth discussing.

Not true, on the contrary, you should thank to the transaction fee, it helps the deflation which leads to the TRX increment in recent months. Compared to that, transaction fee is nothing to me considering staking method.

**Comment by ikirudoughnuts** [comment]
You really think the TRX price can be increased by the high transaction fee? I would only thank to the market, all crypto tokens have risen. You do not understand the cost for dapps involving large amount of transactions perhaps.

**Comment by WalterBrooks** [comment]
> You really think the TRX price can be increased by the high transaction fee? I would only thank to the market, all crypto tokens have risen. You do not understand the cost for dapps involving large amount of transactions perhaps.

No doubt TRX price increases more than ETH. I am not saying that transaction fee is the only reason to prompt price, but we should regard the transaction fee properly. Good energy price incites the activity on chain and meanwhile helps token price increment.

**Comment by vivian1912** [comment]
The unit price of energy has already been decreased to 210 sun, please refer to #668 and [Proposal No.95](https://tronscan.org/#/proposal/95) .
